servers/src/main/java/tachyon/worker/block/evictor/LRUEvictor.java

```diff
@@ -99,20 +126,12 @@ public EvictionPlan freeSpace(long availableBytes, BlockStoreLocation location)
    */
   @Override
   public void onAccessBlock(long userId, long blockId) {
-    if (mLRUCache.containsKey(blockId)) {
-      mLRUCache.get(blockId);
-    } else {
-      mLRUCache.put(blockId, true);
-    }
+    mLRUCache.put(blockId, true);
```

Comment:
nop, it is just there to occupy to value field of map, LinkedHashMap will maintain the access order by itself, no need to sort by value such as access time.